roofit/hs3/src/RooJSONFactoryWSTool.cxx

```diff
@@ -128,6 +128,35 @@ bool RooJSONFactoryWSTool::registerExporter(const TClass *key, const RooJSONFact
    return true;
 }
 
+int RooJSONFactoryWSTool::removeImporters(const std::string &needle)
+{
+   std::vector<std::string> toDelete;
+   for (auto &element : RooJSONFactoryWSTool::_importers) {
+      TString name(typeid(*element.second).name());
+      if (name.Contains(needle)) {
```

Comment:
```suggestion
   for (auto const& element : RooJSONFactoryWSTool::_importers) {
      std::string name = typeid(*element.second).name();
      if (name.find(needle) != std::string::npos) {
```
Avoid using old TStrings if you can use `std::string` instead (and loop element can be `const&`).